concept/connection/GraknConcepts.java

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
We decided we should move this file to the main `connection` package, which breaks a cyclic dependency between `connection` and `concepts`

Reply:
Done